src/main/java/edu/wisc/my/messages/controller/MessagesController.java

```diff
@@ -1,26 +1,30 @@
 package edu.wisc.my.messages.controller;
 
-import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.json.JSONObject;
-import edu.wisc.my.messages.service.MessagesService;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import edu.wisc.my.messages.model.MessageArray;
+import edu.wisc.my.messages.model.Message;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.ResponseBody;
 
+import edu.wisc.my.messages.service.MessagesService;
+
 @Controller
 public class MessagesController {
     protected final Logger logger = LoggerFactory.getLogger(getClass());
     private MessagesService messagesService;
     
     @RequestMapping(value="/messages", method=RequestMethod.GET)
-    public @ResponseBody void getJson(HttpServletRequest request,
+    public @ResponseBody void getAllMessages(HttpServletRequest request,
```

Comment:
Stipulating that I'm in the minority about this and you should all just tell me I'm wrong: 

I object to this method naming convention. This isn't a JavaBean accessor, it shouldn't be named with a `get` prefix. It's just `public @ResponseBody void messages(...)` or `...allMessages(...)` or so.

Reply:
I would vote messages(), but I'm not too strong on this